README.md

```diff
@@ -56,8 +56,31 @@ Include the following the project's `build.gradle`
 ```gradle
 compile 'com.google.cloud.sql:mysql-socket-factory-connector-j-8:1.1.0'
 ```
+#### MySQL R2DBC
 
-#### PostgreSQL
+##### Maven
```

Comment:
Right now this feels a bit weaving - I have to skip over large parts if I only care about r2dbc. Would it be better to split into a JDBC section and a r2dbc section and have the "installing the library" sections beneath that? 

Also, when we move into different folders we can move these READEMEs into those folders and just link from the main README.

Reply:
Splitting into JDBC/R2DBC sections makes sense. I'll reformat this.